app/routes/users.js

```diff
@@ -18,9 +18,9 @@ Router.get(
   UsersController.welcomeUser,
 );
 
-Router.get(
-  '*',
-  UsersController.wrongUrl,
-);
+// Router.get(
```

Comment:
Kindly uncomment this code.

Reply:
I moved this route to `routes/index.js` so that it works for every route. plus it will work for every method `post` , `put` e.t.c